index.html

```diff
@@ -208,16 +208,27 @@ <h5>Get started</h5>
     </div>
 
     <ul class="form-name-box">
-      <li class="form-name-gray">Your name</li>
-      <li class="fill-name">Anastasiia</li>
+      <li class="form-name-gray"><label for="name">Your name</label></li>
+      <li class="fill-name">
+        <input type="text" name="name" id="name" value="Anastasiia" placeholder="Your name" maxlength="30" required>
+        <input>
+      </li>
+
     </ul>
 
     <ul class="form-email-box">
-      <li class="form-email-gray">Your email</li>
-      <li class="fill-email">|</li>
+      <li class="form-email-gray"><label for="text">e-mail</label></li>
+      <li class="fill-email"><input type="text" name="e-mail" id="text" value="" placeholder="|" maxlength="30"
+          required>
+      </li>
     </ul>
+
     <ul class="message-box">
-      <li class="fill-messagge">Enter your message here...</li>
+      <li class="fill-messagge"><label for="text"></label>
```

Comment:
- [ ] You should use `<form>` tag instead of `<ul>` because `<form>` is used to collect data and it has its own unique properties that can't be substituted by `<ul>` or other tags. Please have a look at [this](https://www.w3schools.com/html/html_forms.asp) tutorial on how to create HTML5 forms.
- [ ] You should submit form data to **Formspree** as per the project requirement. Please read [here](https://formspree.io/html) on how to use it.